content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -40,11 +42,11 @@ You can quickly set up Prometheus and Grafana for testing using the Prometheus a
     internal.<cluster_address>:8070
     ``` 
 
-1. Create an instance to run Prometheus and Grafana on the same provider as your Redis Cloud subscription (Amazon Web Services or Google Cloud Project). This instance must:
+1. Create an instance to run Prometheus and Grafana on the same cloud provider as your Redis Cloud subscription (for example, Amazon Web Services or Google Cloud Platform). This instance must:
```

Comment:
@cmilesb Google changes the name of Google Cloud Platform to just "Google Cloud" so any reference to GCP should be changed to the latter.